tools/archetype/binding/src/main/resources/archetype-resources/ESH-INF/binding/binding.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>

```

Comment:
To my understanding, according to https://www.eclipse.org/legal/ _all_ distributed (source) files _must_ have have a license header. So this file should have one too, or am I missing anything? (I know, we need to check the existing codebase too....)